docs/UserGuide.md

```diff
@@ -443,13 +443,17 @@ Format: `exit`
 #### View students and rooms side by side: `toggle-split`
 While allocating rooms to students, it is probably easier to view rooms and students at the same time. If the rooms and students tabs are currently separate, `toggle-split` will merge the students and rooms tab into 1 tab that shows them side by side, which is shown the image below. You can use the `rooms` or `students` commands to switch to the combined tab as usual.
 
+Note that `toggle-split` will cause the application to switch to the combined students and rooms tab, regardless of what tab you previously had open.
```

Comment:
Perhaps these notes can be wrapped in an info modal to stand out, similar to the ones used in certain sections of the UG.

![image](https://user-images.githubusercontent.com/22059033/98390326-28fd4b00-2090-11eb-833c-882f6a15e6b7.png)

Reply:
Oh that's a good suggestion, will do!